JM/CF/Scripts/3_Game/CommunityFramework/Timer/CF_Timer.c

```diff
@@ -0,0 +1,55 @@
+class CF_Timer : CF_TimerBase
+{
+	protected Managed m_Object;
```

Comment:
A better name could be `m_Instance`... `m_Object` would imply a world object created by `GetGame().CreateObject()`